xhr/resources/headers-some-are-empty.asis

```diff
@@ -0,0 +1,6 @@
+HTTP/1.0 200 MEH
+HEYA:	 
+HEYA: 1
+HEYA:
+HEYA:	
```

Comment:
Suggest adding trailing whitespace either after the 1 or 2, but not both (At least I think neither of them has trailing whitespace)